Crash on v1.0.0.8: lateinit property has not been initialized

1. while on camera preview mode, go and revoke all permissions
2. switch back to the app, observe it crashes

```
2019-09-06 08:21:10.332 23181-23181/com.automattic.portkey E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.automattic.portkey, PID: 23181
    java.lang.RuntimeException: Unable to pause activity {com.automattic.portkey/com.automattic.portkey.compose.ComposeLoopFrameActivity}: kotlin.UninitializedPropertyAccessException: lateinit property originalMatrix has not been initialized
        at android.app.ActivityThread.performPauseActivityIfNeeded(ActivityThread.java:3912)
        at android.app.ActivityThread.performPauseActivity(ActivityThread.java:3878)
        at android.app.ActivityThread.performPauseActivity(ActivityThread.java:3852)
        at android.app.ActivityThread.handlePauseActivity(ActivityThread.java:3826)
        at android.app.ActivityThread.-wrap15(Unknown Source:0)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1601)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6494)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
     Caused by: kotlin.UninitializedPropertyAccessException: lateinit property originalMatrix has not been initialized
        at com.automattic.photoeditor.camera.VideoPlayingBasicHandling.stopVideoPlay(VideoPlayingBasicHandling.kt:103)
        at com.automattic.photoeditor.camera.VideoPlayingBasicHandling.windDown(VideoPlayingBasicHandling.kt:124)
        at com.automattic.photoeditor.camera.VideoPlayingBasicHandling.onPause(VideoPlayingBasicHandling.kt:89)
        at androidx.fragment.app.Fragment.performPause(Fragment.java:2777)
        at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManagerImpl.java:933)
        at androidx.fragment.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManagerImpl.java:1238)
        at androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManagerImpl.java:1303)
        at androidx.fragment.app.FragmentManagerImpl.dispatchStateChange(FragmentManagerImpl.java:2656)
        at androidx.fragment.app.FragmentManagerImpl.dispatchPause(FragmentManagerImpl.java:2626)
        at androidx.fragment.app.FragmentController.dispatchPause(FragmentController.java:279)
        at androidx.fragment.app.FragmentActivity.onPause(FragmentActivity.java:420)
        at android.app.Activity.performPause(Activity.java:7153)
        at android.app.Instrumentation.callActivityOnPause(Instrumentation.java:1408)
        at android.app.ActivityThread.performPauseActivityIfNeeded(ActivityThread.java:3901)
        at android.app.ActivityThread.performPauseActivity(ActivityThread.java:3878) 
        at android.app.ActivityThread.performPauseActivity(ActivityThread.java:3852) 
        at android.app.ActivityThread.handlePauseActivity(ActivityThread.java:3826) 
        at android.app.ActivityThread.-wrap15(Unknown Source:0) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1601) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:164) 
        at android.app.ActivityThread.main(ActivityThread.java:6494) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807) 
```

--- a/photoeditor/src/main/java/com/automattic/photoeditor/camera/VideoPlayingBasicHandling.kt
+++ b/photoeditor/src/main/java/com/automattic/photoeditor/camera/VideoPlayingBasicHandling.kt
@@ -65,7 +65,7 @@ class VideoPlayingBasicHandling : Fragment(), SurfaceFragmentHandler, VideoPlaye
      * An [AutoFitTextureView] for camera preview.
      */
     lateinit var textureView: AutoFitTextureView
-    private lateinit var originalMatrix: Matrix
+    lateinit var originalMatrix: Matrix
 
     private var active: Boolean = false
 

--- a/photoeditor/src/main/java/com/automattic/photoeditor/state/BackgroundSurfaceManager.kt
+++ b/photoeditor/src/main/java/com/automattic/photoeditor/state/BackgroundSurfaceManager.kt
@@ -334,6 +334,7 @@ class BackgroundSurfaceManager(
                     videoPlayerHandling = videoPlayerFragment as VideoPlayingBasicHandling
                     // the photoEditorView layout has been recreated so, re-assign its TextureView
                     videoPlayerHandling.textureView = photoEditorView.textureView
+                    videoPlayerHandling.originalMatrix = photoEditorView.textureView.getTransform(null)
                 }
                 // add video player texture listener
                 photoEditorView.listeners.add(videoPlayerHandling.surfaceTextureListener)

